recipe/meta.yaml

```diff
@@ -1,34 +1,29 @@
-{% set maturin_version = ">=1.2.3,<2" %}
-{% set maturin_version = ">=1.2.3,<2" %}
-{% set maturin_version = ">=1.2.3,<2" %}
-{% set maturin_version = ">=1.2.3,<2" %}
-{% set maturin_version = ">=1.2.3,<2" %}
-{% set version = "0.8.1" %}
+{% set version = "0.8.2" %}
 
 package:
   name: pycrdt
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/pycrdt/pycrdt-{{ version }}.tar.gz
-  sha256: cdcce2a883b01ac43b4e00687319699b274f9dcfb68cd29ae004cc5f08dbcff3
+  sha256: 50817c55c8c4dd0f6df0f67f34c34bedc3fba440019b1e8d87b0e0579b6d79d8
 
 build:
   number: 0
   script:
-    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+    - {{ PYTHON }} -m pip install . -vv
     - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
```

Comment:
Why remove the selector?

Reply:
I was trying to copy what's done in [watchfiles](https://github.com/conda-forge/watchfiles-feedstock/blob/daab0c33a93e53b23cdec9f571bde5278ebcd36f/recipe/meta.yaml), because it's a similar package (mixed Python/Rust) and it works there, but the issue happens without that change. I'll revert it to check.